### DISCUSSION TITLE
How to disable default mappings ?

### DISCUSSION THREAD

**Discussion by mb-915** [folke/trouble.nvim#570] [Q&A]
Is there a way to disable one or all default mappings ?

I want to remove the `s` mapping because I use it for something else.

I can override it using 
```lua
keys = {
  s = { action = … },
},
```
But I wanted to unmap it so that my existing mapping applies without rewriting it. 

Is there a parameter or a way to disable a default mapping ?

**Comment by winston0410** [comment]
@mb-915 did you find a way to remove a default mapping at the end?

**Reply by mb-915** [reply]
No 😕

I used `s` to go down, so I ended up rewriting the keybinding using `vim.api`.
Later, I also stopped using `s` to go down (and switched to using the default `hjkl` instead of my custom remap), and do not need this feature anymore (because I was tired of remaping in every software and plugin incompatibilities 🙃)
